src/app/routes/index.js

```diff
@@ -1,16 +1,24 @@
-import { Route, Switch } from 'react-router-dom';
 import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import PropTypes from 'prop-types';
 import ChannelsListContainer from '../features/Chanels/containers/ChannelsListContainer';
 import ChannelPageContainer from '../features/Chanels/containers/ChannelPageContainer';
 import 'antd/dist/antd.css';
+import LogInPage from '../components/LologInPage';
 
-const RoutesResolver = () => {
-  return (
+const RoutesResolver = ({ authorized }) => {
+  return authorized ? (
     <Switch>
       <Route exact path="/channels" component={ChannelsListContainer} />
       <Route path="/channels/:id" component={ChannelPageContainer} />
     </Switch>
+  ) : (
```

Comment:
Also, please, take a look at this example and do the same: 

https://reacttraining.com/react-router/web/example/auth-workflow

The idea is to have separate route for login screen